tests/partial_application/partial_application.fz

```diff
@@ -59,17 +59,19 @@ partial_application is
   test "data.map 3.infix*  " (data.map    3.infix*  ) "[3, 6, 9, 12, 15, 18, 21, 24, 27, 30]"
   test "data.map 4.infix*  " (data.map    4.infix*  ) "[4, 8, 12, 16, 20, 24, 28, 32, 36, 40]"
 
-  i32.prefix      !! => if val <= 1 then 1 else val * !!(val-1)
-  integer.postfix !  => if integer.this <= integer.this.one then integer.this.one else integer.this * (integer.this-integer.this.one)!
-  i32.postfix !  => if val <= 1 then 1 else val * (val-1)!
+  i32.prefix      !!  => if val <= 1 then 1 else val * !!(val-1)
+  integer.postfix !   => if integer.this <= integer.this.one then integer.this.one else integer.this * (integer.this-integer.this.one)!
+  i32.postfix !!!     => if val <= 1 then 1 else val * (val-1)!!!
   u64.postfix     !!  => if val <= u64 1 then u64 1 else val * (val-1)!!
   u64.prefix      !!  => if val <= u64 1 then u64 1 else val * !!(val-1)
 
   test "data.map (+)" (data.map (+)) "[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]"
   test "data.map (-)" (data.map (-)) "[-1, -2, -3, -4, -5, -6, -7, -8, -9, -10]"
   test "data.map (!!)" (data.map (!!)) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
-  test "data.map (!)" (data.map (!)) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
-  test "data.map (!) " (data.map (!) ) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
+  # NYI should use integer.postfix ! but does not find feature currently
+  # test "data.map (!)" (data.map (!)) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
+  test "data.map (!!!)" (data.map (!!!)) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
+  test "data.map (!!!) " (data.map (!!!) ) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
   test "((u64 11)..20).map (!)" (((u64 11)..20).map (!)) "[39916800, 479001600, 6227020800, 87178291200, 1307674368000, 20922789888000, 355687428096000, 6402373705728000, 121645100408832000, 2432902008176640000]"
   test "((u64 11)..15).map ( !)" (((u64 11)..15).map ( !)) "[39916800, 479001600, 6227020800, 87178291200, 1307674368000]"
   test "((u64 11)..15).map (! )" (((u64 11)..15).map (! )) "[39916800, 479001600, 6227020800, 87178291200, 1307674368000]"
```

Comment:
This seems unrelated.

Reply:
@fridis 
With this patch, an error would be raised for duplicate feature because of:
integer.postfix !
i32.postfix !